web/templates/index.html

```diff
@@ -92,9 +98,12 @@ <h6 class="card-subtitle mb-2 text-muted">
                                     </div>
                                     <div class="card-text">
                                         <select class="form-control" id="lang" name="lang">
-                                                {% for lang in languages %}
-                                                <option>{{ lang }}</option>
-                                                {% endfor %}
+                                            <option selected>{{exampleLanguages.0}}</option>
+                                            {% for lang in languages %}
+                                                {% if lang != exampleLanguages.0 %}
+                                                    <option>{{ lang }}</option>
+                                                {% endif %}
+                                            {% endfor %}
```

Comment:
Same thing here (move selected into the for loop). Also did you copy and paste without changing `0` to `2`? 😁

Reply:
Nice catch!